packages/cli/src/lib/SchemaComposer.ts

```diff
@@ -0,0 +1,152 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/naming-convention */
+/* eslint-disable @typescript-eslint/no-empty-function */
+
+import { Web3ApiManifest } from "./Web3ApiManifest";
+import { displayPath } from "./helpers/path";
+import { withSpinner } from "./helpers/spinner";
+import { BuildConfig } from "./Compiler";
+
+import fs from "fs";
+import path from "path";
+import { Manifest, Uri, Web3ApiClient, UriRedirect } from "@web3api/client-js";
+import { composeSchema, ComposerOutput } from "@web3api/schema-compose";
+import { EnsPlugin } from "@web3api/ens-plugin-js";
+import { EthereumPlugin } from "@web3api/ethereum-plugin-js";
+import { IpfsPlugin } from "@web3api/ipfs-plugin-js";
+
+export class SchemaComposer {
+  constructor(private _config: BuildConfig) {}
+
+  public async loadManifest(quiet = false): Promise<Manifest> {
```

Comment:
This can be made into its own helper function that takes the `manifestPath: string` as an argument. This will be helpful in making these classes smaller. I also don't think it makes sense to have "loadManifest" as a method on the SchemaComposer, seems like this mixes responsibilities a bit.